src/controllers/userController.js

```diff
@@ -8,6 +8,18 @@ const {
 } = require('../utils');
 
 const getAllUsers = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch all registered users in a database whose role is user (exclude admins)'
+     #swagger.parameters['role'] = {
+        description: 'Role of the user to fetch',
+     }
+     #swagger.responses[200] = {
+				description: 'Users successfully fetched.',
+        schema: [{ $ref: '#/definitions/PasswordlessUser'' }]
```

Comment:
Actually, the endpoint doesn't return an array directly. When I use postman to test this endpoint I get an object with the key `users` and the value of that is the array of users:

```
{ 
  "users": [ array of users here ]
}
```

So we should make that clear to the reader of these api docs:

```suggestion
        schema: { users: [{ $ref: '#/definitions/PasswordlessUser' }] }
```

Reply:
@akosasante I am trying to test this getAllUsers route and get in postman
![Captura de pantalla 2023-08-29 a las 0 10 40](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/103755090/084dd44c-fd75-435e-b006-62509b8c65b2)
//maybe i am testing the wrong route- where do you get ```
{ 
  "users": [ array of users here ]
}
``` ?

Reply:
To clarify, I'm referring to the response body. For example, in your screenshot the JSON returned is an object, with a key of `"users"` and then the list of users. Which is to say, it returns an object and not just directly the list; otherwise we would see in the response body:

```
[
  { "_id": blah, name: blah, etc},
  { "_id": blah2, name: blah2, etc}
]
```

But instead we get back what you have there which is:

```
{
  "users": [
    { "_id": blah, name: blah, etc},
    { "_id": blah2, name: blah2, etc},
  ]
}
```

We need our documentation to describe that as well so that on the frontend they'll know that they have to do something like `response.data.users` instead of `response.data`

Reply:
@akosasante got it! thank you!!